dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -64,18 +64,25 @@ public CultureInfo Culture
     /// </summary>
     public IAIServiceProvider ServiceProvider { get; }
```

Comment:
The service provided may no longer need to be available through the context, right?

Reply:
Yes, I'm going to make this internal